app/generator/species_generator.rb

```diff
@@ -0,0 +1,8 @@
+class SpeciesGenerator
+  def self.save_species
+    all_trails = Trail.all.limit(3)
+    all_trails.each do |trail|
+      ApiCombiner.get_species(trail.id)
+    end
```

Comment:
Finding species upto 20 in the first 3 trails...make sense with all the trails we have will take a long time.